packages/core-components/src/components/Avatar/Avatar.tsx

```diff
@@ -29,7 +30,7 @@ const useStyles = makeStyles(
       avatar: {
         width: '4rem',
         height: '4rem',
-        color: '#fff',
+        color: grey[50],
```

Comment:
Should probably just be `white` instead right?

Reply:
Nope, there are articles against using pure colors, pure white is harder on the eyes, I think that's why there's no `grey[0]`, not sure if that's the same reason there's no `grey[1000]` for pure black...

Reply:
There's `common.white` and `common.black` to represent `#fff` and `#000`

Reply:
Basically it's about contrast, pure white & pure black make for too high contrast, so it strains the eye, that's why you see the themes of most professional websites using almost white colors like `#fdfdfd` for backgrounds and `#2f2f2f` for text color, would be nice to get confirmation from an actual designer...

Reply:
Didn't know it existed in the theme, but now I do, but the point still stands, as far as color design goes we should avoid pure white & pure black

Reply:
This is why I would ask a designer if possible, so should I just go for `common` colors in these cases?

Reply:
Yep I think so. It's easier to do the broad refactor if we avoid actual visual changes, then individual tweaks could be handled separately with a more focused discussion

Reply:
Swapped all whites & blacks to `theme.palette.common.{color}`